TestPlaybooks/playbook-Git_Integration-Test.yml

```diff
@@ -1259,8 +1260,149 @@ tasks:
+          "x": 490,
+          "y": 1945
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "48":
+    id: "48"
+    taskid: 42238f9a-622d-4a1e-8d5d-5d6e4c381bbd
+    type: regular
+    task:
+      id: 42238f9a-622d-4a1e-8d5d-5d6e4c381bbd
+      version: -1
+      name: GitHub-is-pr-merged
+      description: 'Get if a pull request has been merged. If the pull request has
+        been merged the API returns ''Status: 204 No Content''. If the pull request
+        has not been merged the API returns ''Status: 404 Not Found'''
+      script: '|||GitHub-is-pr-merged'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "49"
+    scriptarguments:
+      pull_number:
+        simple: "82"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 1012.5,
+          "y": 1770
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "49":
+    id: "49"
+    taskid: 2c21fbd5-fd5a-4951-8894-146d3aa61928
+    type: regular
+    task:
+      id: 2c21fbd5-fd5a-4951-8894-146d3aa61928
+      version: -1
+      name: GitHub-create-pull-request
+      description: Create a new pull request
+      script: '|||GitHub-create-pull-request'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "50"
+    scriptarguments:
+      base:
+        simple: master
+      body:
+        simple: CREATED BODY
+      draft: {}
+      head:
+        simple: branch-for-pr
+      maintainer_can_modify: {}
+      title:
+        simple: CREATED PR
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 1012.5,
+          "y": 1945
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "50":
+    id: "50"
+    taskid: 891d0bf7-f6d0-4853-887e-202f920c15b7
+    type: regular
+    task:
+      id: 891d0bf7-f6d0-4853-887e-202f920c15b7
+      version: -1
+      name: GitHub-update-pull-request
+      description: Update a pull request
+      script: '|||GitHub-update-pull-request'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "51"
+    scriptarguments:
+      base: {}
+      body:
+        simple: UPDATED PR
+      maintainer_can_modify: {}
+      pull_number:
+        complex:
+          root: GitHub
+          accessor: PR.Number
+      state: {}
+      title:
+        simple: UDPATE PR
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 1012.5,
+          "y": 2120
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "51":
+    id: "51"
+    taskid: 00411238-5332-4a97-85d3-78faef569396
+    type: regular
+    task:
+      id: 00411238-5332-4a97-85d3-78faef569396
+      version: -1
+      name: GitHub-close-issue
+      description: Closes an existing issue.
+      script: '|||GitHub-close-issue'
```

Comment:
we are not verifying it was closed

Reply:
no need. This was really just for utility - I didn't want to leave a bunch of opened PRs. The close issue command is actually executed and checked elsewhere in the playbook.